google/cloud/bigtable/client_options_test.cc

```diff
@@ -37,32 +37,12 @@ struct ClientOptionsTestTraits {
 namespace {
 
 using ::testing::HasSubstr;
+// NOLINTNEXTLINE(readability-identifier-naming)
+auto const GetInt = ::google::cloud::internal::GetIntChannelArgument;
```

Comment:
Rather than declaring these here, I think it would be clearer if this file just included `google/cloud/grpc_options.h` and then directly called the new functions with their new names.

Reply:
Done.